core/src/main/scala/Interval.scala

```diff
@@ -0,0 +1,36 @@
+package helm
+
+import scalaz._, Scalaz._
+import argonaut._, Argonaut._
+
+sealed abstract class Interval
+
+/** Some types to represent check intervals, et al. */
+object Interval {
+
+  // So I considered using Scala's FiniteDuration type, but it
+  // seems like overkill.
```

Comment:
I'm open to alternative implementations for this, if you have something else in mind. I think using the type system to enforce properly formed interval values is preferable, but beyond that I'm not married to any particular design.